docs/index.md

```diff
@@ -87,33 +93,31 @@ Install Codacy on an existing cluster using our Helm chart:
                  # --values values-microk8s.yaml
     ```
 
-7.  By now all the Codacy pods should be starting in the cluster. Run the following command **and wait for all the pods to have the status Running** to verify this:
+7.  By now all the Codacy pods except `codacy-crow` (which can be [configured later](configuration/monitoring.md#setting-up-monitoring-using-crow)) should be starting in the cluster.
```

Comment:
The crowdb should be configured in the values-production.yaml and start right away.

@joaocosta-codacy I think we have a bug here.

Reply:
I am looking into it.

Reply:
https://github.com/codacy/chart/pull/371

Reply:
Wow, this is great! I'll revert these changes, then. :+1: